packages/core/core/src/factories/apiFactory/types.ts

```diff
@@ -0,0 +1,51 @@
+import { IntegrationPlugin } from './../../utils/nuxt';
```

Comment:
just please check this https://github.com/vuestorefront/vue-storefront/pull/5361/files#diff-94593f73bba756dabad3c9eaca20619745bbae240f966b5fceb7802194df1dd0 and adjust the file ;)